### PR TITLE
[JBTM-3307] enabling LRACoordinatorRecovery2TestCase with timeout adjustments

### PR DESCRIPTION
**Pull request by ochaloup**
https://issues.redhat.com/browse/JBTM-3307

!MAIN !CORE !QA_JTA !QA_JTS_JDKORB !QA_JTS_OPENJDKORB !QA_JTS_JACORB !BLACKTIE !XTS !PERF NO_WIN !RTS !AS_TESTS !TOMCAT !JACOCO
LRA

### CODE REVIEW

File: rts/lra/coordinator/pom.xml [hunk 1]
```diff
@@ -163,6 +163,7 @@
                                 <lra.coordinator.host>${lra.coordinator.host}</lra.coordinator.host>
                                 <lra.coordinator.url>${lra.coordinator.url}</lra.coordinator.url>
                                 <server.jvm.args>${server.jvm.args} -Dlra.coordinator.url=${lra.coordinator.url}</server.jvm.args>
+                                <lra.tck.timeout.factor>${lra.tck.timeout.factor}</lra.tck.timeout.factor>
```
Comment (xstefank) [inline rts/lra/coordinator/pom.xml:166]: maybe it shouldn't be named with `tck` in the name.

**Review by xstefank — APPROVED** [review]

**Comment by ochaloup** [comment]
There is a failure which I haven't hit during my investigation (I run LRA several times with AMS CI with my changes).
It's  with `LRACoordinatorRecovery1TestCase.testRecovery` and I haven't seen it yet. I need to investigate.

File: rts/lra/coordinator/pom.xml (hunk 1 above)
Comment (ochaloup) [reply rts/lra/coordinator/pom.xml:166]: That's a good point. I was thinking if it's good to use it with the name `tck` in it but then I leave it as it is as it's how it's used.
I feel it's not a great approach. I will try to come with a general name.

**Review by mmusgrov — APPROVED** [review]

File: rts/lra/coordinator/pom.xml (hunk 1 above)
Comment (ochaloup) [reply rts/lra/coordinator/pom.xml]: @xstefank I changed the naming. If you have a while, please, take a look.

**Comment by ochaloup** [comment]
The trouble with `LRACoordinatorRecovery1TestCase.testRecovery` was the fact the both tests are running as client side and the first test does not clean the `LRAClient` context from the started LRAs and the following test was trying to join the cancelled LRA. Hopefully fixed now.

**Review by xstefank — APPROVED** [review]
LGTM. Can be merged.

**Pull request merged**